packages/react-components/src/components/RTE/RTESendBox.tsx

```diff
@@ -19,6 +31,25 @@ export interface RTESendBoxProps {
  * @beta
  */
 export const RTESendBox = (props: RTESendBoxProps): JSX.Element => {
-  const { valueToDisplay = 'Hello World!' } = props;
-  return <div>{valueToDisplay}</div>;
+  const { disabled } = props;
+
+  const theme = useTheme();
+  const localeStrings = useLocale().strings.sendBox;
+  const strings = { ...localeStrings, ...props.strings };
+
+  const [textValue] = useState('');
```

Comment:
would textValue be confused when it's actually htmlValue?

Reply:
![image](https://github.com/Azure/communication-ui-library/assets/73612854/96aacc97-a6ff-411d-970d-ed64c9cf2bca)

I could match it with the richTextEditor props. Make it 'content' instead.

Reply:
Updated